serde-generate/serde-generate-bin/tests/cli.rs

```diff
@@ -1,10 +1,123 @@
 // Copyright (c) Facebook, Inc. and its affiliates
 // SPDX-License-Identifier: MIT OR Apache-2.0
 
-use serde_generate::test_utils;
 use std::process::Command;
 use tempfile::tempdir;
 
+mod test_utils {
```

Comment:
The alternative was to use a feature `test` in `serde-generate` but this may be overkill.